browser/resources/settings/sources.gni

```diff
@@ -61,6 +61,9 @@ brave_settings_web_component_files = [
 ]
 
 brave_settings_non_web_component_files = [
+  "email_aliases_page/email_aliases_page.ts",
+  "email_aliases_page/content/remote_mapping_service.ts",
+  "email_aliases_page/content/types.ts",
```

Comment:
We can remove these lines if we create the remote mapping service from `email_aliases.tsx`

Reply:
fixed